targets/TARGET_STM/gpio_object.h

```diff
@@ -61,7 +61,7 @@ static inline void gpio_write(gpio_t *obj, int value)
         *obj->reg_set = obj->mask;
     } else {
 #ifdef GPIO_IP_WITHOUT_BRR
-        *obj->reg_clr = obj->mask << 16;
+        *obj->reg_set = obj->mask << 16;
 #else
```

Comment:
This part of the patch should not be done / is not required because this is managed in gpio_api.c where the reg_clr register is defined as the same BSRR register, here
https://github.com/ARMmbed/mbed-os/blob/master/targets/TARGET_STM/gpio_api.c#L142